src/main/java/org/folio/search/service/converter/preprocessor/InstanceEventPreProcessor.java

```diff
@@ -24,21 +25,28 @@
 import org.folio.search.utils.JsonConverter;
 import org.springframework.stereotype.Service;
 
+@Log4j2
 @Service
 @RequiredArgsConstructor
 public class InstanceEventPreProcessor implements EventPreProcessor {
 
   public static final String SUBJECTS_FIELD = "subjects";
 
-  private static final TypeReference<List<SubjectResourceEvent>> TYPE_REFERENCE = new TypeReference<>() { };
+  private static final TypeReference<List<SubjectResourceEvent>> TYPE_REFERENCE = new TypeReference<>() {
+  };
 
   private final JsonConverter jsonConverter;
 
   @Override
   public List<ResourceEvent> process(ResourceEvent event) {
+    log.debug("process:: by [event: {}]", event);
+
     var oldSubjects = extractSubjects(getOldAsMap(event));
     var newSubjects = extractSubjects(getNewAsMap(event));
     var tenantId = event.getTenant();
+
+    log.info("process:: Attempting create list of resource event by [newSubjects: {}, oldSubjects: {}, tenant: {}]",
```

Comment:
```suggestion
    log.debug("process:: Attempting create list of resource event by [newSubjects: {}, oldSubjects: {}, tenant: {}]",
```
There could be a very long lists